payone-adapter/src/test/java/com/commercetools/sunrise/payment/payone/PayonePaymentServiceProviderImplTest.java

```diff
@@ -0,0 +1,61 @@
+package com.commercetools.sunrise.payment.payone;
+
+import com.commercetools.sunrise.payment.domain.PaymentServiceProvider;
+import io.sphere.sdk.payments.PaymentMethodInfo;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by mgatz on 7/18/16.
+ */
+public class PayonePaymentServiceProviderImplTest {
+
+    private static final String METHOD_NAME_CC = "PayoneCC";
+    private static final String METHOD_NAME_PAYPAL = "PayonePP";
+    private static final String METHOD_NAME_SOFORT = "PayoneSB";
+    public static final String METHOD_ID = "Payone";
+
+    @Test
+    public void getId() throws Exception {
+        assertThat(new PayonePaymentServiceProviderImpl().getId(), is(METHOD_ID));
+    }
+
+    @Test
+    public void getAvailablePaymentMethods() throws Exception {
+        PaymentServiceProvider psp = new PayonePaymentServiceProviderImpl();
+        assertThat(psp.getAvailablePaymentMethods().size(), is(3));
+
+        PaymentMethodInfo pmi = psp.getAvailablePaymentMethods().get(0); // credit card
+        assertThat(pmi.getPaymentInterface(), is(METHOD_ID));
```

Comment:
For Sunrise we always use [AssertJ](http://joel-costigliola.github.io/assertj/) for assertions. It is very convenient with Java 8 and quite easy to read and write. It would be good if we can use them here too.